src/xcode/ENA/ENA/Source/Scenes/HealthCertificates/HealthCertifiedPerson/ReissuanceConsent/HealthCertifiedPersonReissuanceConsentViewController.swift

```diff
@@ -4,18 +4,28 @@
 
 import UIKit
 
-class HealthCertifiedPersonReissuanceConsentViewController: UIViewController, DismissHandling, FooterViewHandling {
+class HealthCertifiedPersonReissuanceConsentViewController: DynamicTableViewController, DismissHandling, FooterViewHandling {
```

Comment:
I think the name should be either `CertificateReissuanceConsentViewController` or `HealtCertificateReissuanceConsentViewController` as it's the certificate that is being reissued, and it would be in line with the model then.

Reply:
Solved